OpenRA.Mods.Common/Traits/Crates/HideMapCrateAction.cs

```diff
@@ -13,13 +13,21 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Hides the entire map in shroud.")]
 	class HideMapCrateActionInfo : CrateActionInfo
 	{
+		[Desc("Should the map also be hidden for the allies of the collector's owner.")]
+		public readonly bool IncludeAllies = false;
+
 		public override object Create(ActorInitializer init) { return new HideMapCrateAction(init.Self, this); }
 	}
 
 	class HideMapCrateAction : CrateAction
 	{
+		readonly HideMapCrateActionInfo info;
+
 		public HideMapCrateAction(Actor self, HideMapCrateActionInfo info)
-			: base(self, info) { }
+			: base(self, info)
+		{
+			this.info = info;
```

Comment:
Leave it. If you use `Info` you'd likely need to cast it to use it, and that makes it not worth the trouble.